src/elements/form/extras/address/Address.js

```diff
@@ -73,17 +82,25 @@ export class NovoAddressElement implements ControlValueAccessor {
 
     updateControl() {
         this.onModelChange(this.model);
+        this.changeDetectorRef.markForCheck();
     }
 
     writeValue(model:any):void {
-        if (!this.model || !this.model.length) {
-            this.model = {
-                countryID: 1,
-                countryName: 'United States'
-            };
-        } else {
-            this.model = model;
-        }
+        setTimeout(() => {
```

Comment:
Yes to the check. That error only occurs because you do the state stuff I think. If you were to use `this.model = Object.assign({ countryName, state }, model)` and not call into `this.onStateChange` it should not occur.